jquery.flot.navigate.js

```diff
@@ -176,7 +175,6 @@ can set the default in the options.
             plot.getPlaceholder().css('cursor', plot.getOptions().pan.cursor);
             startPageX = e.pageX;
             startPageY = e.pageY;
-            plotState = plot.navigationState();
```

Comment:
this is important for smart pan as it does all the computation relative to the initial state